app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<body>
+  <h2>Get Shit Done:</h2>
+  <ul class="whole-list">
+    <% @tasks.each do |task|%>
+    <li class="task-list">
+      <% if task.completed != "No" %>
+
+
+      <span class="each">
+        <%= link_to task.task, task_path(task[:id]), class: "done" %> |
+        <%= link_to "UnMark Complete", mark_completed_path(task.id), method: :patch, class: "option"%> |
```

Comment:
I like how you have this structured to toggle between complete and incomplete.  Very thoughtful.  The styling is nice too!